src/components/molecules/ChannelGuidance/ChannelGuidance.tsx

```diff
@@ -0,0 +1,29 @@
+import { TemplateType } from '@utils/enum';
+import { channelGuidanceContent } from '@content/content';
+import Link from 'next/link';
+
+export type TemplateTypeNoLetters = Exclude<TemplateType, TemplateType.LETTER>;
+
+export type ChannelGuidanceType = {
+  template: TemplateTypeNoLetters;
+};
+
+export function ChannelGuidance({ template }: ChannelGuidanceType) {
+  return (
+    <>
+      <h2 className='nhsuk-heading-m'>
+        {channelGuidanceContent[template].heading}
+      </h2>
+
+      <ul style={{ listStyleType: 'none', padding: 0 }}>
+        {channelGuidanceContent[template].guidanceLinks.map((guidanceLink) => (
+          <li className='nhsuk-u-margin-bottom-3' key={guidanceLink.text}>
+            <Link href={guidanceLink.link} target='_blank'>
```

Comment:
I don't think we are supposed to be using NextJS Links when linking to internal docs. 

When the whole app is deployed to the same domain `https://notify.nhs.uk/using-nhs-notify/delivery-times` will be the same as `/using-nhs-notify/delivery-time`

Reply:
Follow the pattern for the accessibility link in the footer.

Reply:
thanks for pointing out. Updated 👍